### PR TITLE
Kafka tutorial notebook

### PR DESCRIPTION
**Pull request by kvignesh1420**
This PR adds a notebook which provides a tutorial on using `tensorflow_io` package with kafka for machine learning use cases.

### CODE REVIEW

**Comment by kvignesh1420** [comment]
@yongtang I decided to use the https://archive.ics.uci.edu/ml/datasets/SUSY# dataset for the tutorial,  from UCI ML dataset website. Hope it's fine from the licensing aspect.

**Comment by yongtang** [comment]
@kvignesh1420 There are tutorials that uses similar datasets in UCI before so it should be fine. From the link https://archive.ics.uci.edu/ml/datasets/SUSY#  it looks there is citation request, so as long as the tutorial includes the reference it should be good.

**Comment by kvignesh1420** [comment]
@yongtang @terrytangyuan @BryanCutler , please let me know your suggestions regarding this first draft. Thanks

File: docs/tutorials/kafka.ipynb [hunk 1]
```diff
@@ -0,0 +1,1207 @@
+{
```
Comment (BryanCutler) [inline docs/tutorials/kafka.ipynb:1]: <p>Is the accuracy low and not improving per epoch because of the small fraction of training data used? You might want to mention that and run less epochs if that's the case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (kvignesh1420) [reply docs/tutorials/kafka.ipynb:1]: added the comments in the commit

**Comment by yongtang** [comment]
@kvignesh1420 Can you remove the output sections? When the notebook is being pushed to tensorflow.org/io the output will be automatically populated. You can click `X` on the left side of the output sections and save the file.

File: docs/tutorials/kafka.ipynb [hunk 2]
```diff
@@ -0,0 +1,1332 @@
...
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "FONyXboRYe51",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        ""
+      ],
```
Comment (yongtang) [inline docs/tutorials/kafka.ipynb:1327]: Can you also remove this section? It seems this section is empty.

Comment (kvignesh1420) [reply docs/tutorials/kafka.ipynb]: removed in the new commit

**Comment by kvignesh1420** [comment]
@yongtang please check now.

**Comment by kvignesh1420** [comment]
> Looks good in general, thanks for doing this @kvignesh1420 ! I think using the UCI ML dataset is fine, I saw you included a reference to it, maybe also include the web link somewhere.

@BryanCutler added the reference for the dataset.

**Comment by yongtang** [comment]
@kvignesh1420 We are also running lint and fit for doc files. (It is from installing tensorflow/docs with python).

See https://github.com/tensorflow/io/blob/master/.github/workflows/build.yml#L29-L42

For details on how lint/fmt are done for docs.

Can you run the fmt/lint for docs as well?

**Comment by kvignesh1420** [comment]
@yongtang , the lint errors are fixed now. The failed CI step is unrelated.

File: docs/tutorials/kafka.ipynb [hunk 3]
```diff
@@ -0,0 +1,821 @@
...
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "nC-yt_c9u0sH"
+      },
+      "outputs": [],
+      "source": [
+        "susy_df = pd.read_csv('SUSY.csv', header=None)\n",
```
Comment (MarkDaoust) [inline docs/tutorials/kafka.ipynb:441]: Optional:

-You can read the csv right out of the gzip file, and skip the unzip step.
-Instead of loading the whole thing and throwing away most of it, you could just load one chunk:


```
susy_iterator = pd.read_csv('SUSY.csv.gz',
                            header=None, names=COLUMNS,
                            # setting chunksize returins an an iterator
                            chunksize=100000)
susy_df = next(susy_iterator)
susy_df.head()
```

**Comment by kvignesh1420** [comment]
@MarkDaoust  I have made the requested changes. Please review. Thanks.

**Comment by kvignesh1420** [comment]
@yongtang

**Comment by MarkDaoust** [comment]
https://www.tensorflow.org/io/tutorials/kafka

**Comment by yongtang** [comment]
@MarkDaoust nice, thanks a lot! 👍 🎉 🎆
